meeshkan/serve/mock/specs.py

```diff
@@ -16,27 +18,51 @@ class OpenAPISpecification:
     source: str
 
 
-def load_specs(specs_dir: str) -> Sequence[OpenAPISpecification]:
-    if not os.path.exists(specs_dir):
-        logging.info("OpenAPI schema directory not found %s", specs_dir)
-        return []
+def load_specs(specs: Union[str, Sequence[str]]) -> Sequence[OpenAPISpecification]:
+    if isinstance(specs, str):
+        specs = (specs,)
+
     specs_with_sources = []
-    specs_paths = [
-        s
-        for s in os.listdir(specs_dir)
-        if s.endswith("yml") or s.endswith("yaml") or s.endswith("json")
-    ]
-    for spec_path in specs_paths:
-        full_spec_path = os.path.join(specs_dir, spec_path)
-        with open(full_spec_path, encoding="utf8") as spec_file:
+
+    for spec in specs:
+        is_http = spec.startswith("http://") or spec.startswith("https://")
+
+        if is_http or os.path.isfile(spec):
+            specs_paths = [spec]
+        elif os.path.isdir(spec):
+            specs_paths = [
+                os.path.join(spec, dir_entry)
+                for dir_entry in os.listdir(spec)
+                if dir_entry.endswith("yml")
+                or dir_entry.endswith("yaml")
+                or dir_entry.endswith("json")
+            ]
+        else:
+            raise ClickException(f"OpenAPI specification {spec} not found")
```

Comment:
Hmm I think it's a bit suprising to have raise `ClickException` here deep in our code as it introduces a coupling to Click? Maybe just raise an Exception and validate the input in `__main__.py` directly?

Reply:
Done!